gnoi/os/manager_test.go

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package os
+
+import "testing"
+
+func TestIsRunning(t *testing.T) {
+	t.Run("Is running after set running", func(t *testing.T) {
+		manager := NewManager("new")
+		manager.runningVersion = "new"
+		if !manager.IsRunning("new") {
+			t.Error("OS not running after being set running")
+		}
+	})
+	t.Run("Is not running", func(t *testing.T) {
+		manager := NewManager("new")
+		if manager.IsRunning("new") {
+			t.Error("OS running but not set running")
+		}
+	})
+}
```

Comment:
again not tested but, is there any way you could make the following work? I am not specifically against repetitive code especially if it's clean but I guess in this case this can be shortened. Let me know otherwise:

```suggestion
func TestIsRunning(t *testing.T) {
       mc := NewManager("new")
       tests = tests := []struct {
		name string
		mgrClient *Manager
		runningVersion string
		want bool
	}{
		{
			name: "false"
			mgrClient: mc,
			rVer: "new"
			want: false,
		},
	        {
	                name: "true"
			mgrClient: mc,
			rVer: nil,
			want: true,
	        },	
	}
	for _, test := range tests{
		t.Run(t.name, func(t *testing.T) {
			manager.runningVersion = t.rVer
			got := manager.IsRunning("new")
			if  got != t.want {
				t.Errorf("want %s got %v", want, got)
			}
		})
	}
}
```

PS. I am not sure you can use one client for both tests, if not just init a new one every time np.